.changeset/swift-eels-change.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Is "patch" correct?

Reply:
I think a _minor_ version bump is warranted. According to https://semver.org/, you do a "MINOR version when you add functionality in a backwards compatible manner".